test/libyul/yulOptimizerTests/rematerialiser/for_continue_with_assignment_in_post.yul

```diff
@@ -0,0 +1,39 @@
+{
+	let a
+	let b
+	for {let i := 0} lt(i, 10) {i := add(a, b) b := origin()}
```

Comment:
Hm, unfortunately, I don't remember which aspect this was supposed to test anymore..

Reply:
@chriseth the motivation behind this, yesterday, was, to test for loops with continue statements modifying a used variable in the post-block (`b := origin()`). Maybe I should have chosen a different value?